conf/messages.cy

```diff
@@ -1487,6 +1487,9 @@ incomeSources.add.error.incomeSourceNotSaved.incomeSources       = Eich ffynonel
 
 incomeSources.add.error.reportingMethodNotSaved.p1                              = Ychwanegwyd eich busnes {0}, ond nid oedd modd i ni gadw’r manylion ynghylch amlder cyflwyno adroddiadau ar gyfer un neu fwy o flynyddoedd treth.
 incomeSources.add.error.reportingMethodNotSaved.p2                              = Ar gyfer y blynyddoedd treth lle nad oedd modd i ni gadw’r dull o gyflwyno adroddiadau, bydd y manylion ynghylch amlder cyflwyno diweddariadau o ran incwm a threuliau yn newid i ’blynyddol’ yn ddiofyn. Gallwch newid hyn yn nes ymlaen yn yr ’adran i weld a rheoli ffynonellau incwm’.
```

Comment:
The welsh for p1 and p2 don't match with the english

<img width="668" height="459" alt="image" src="https://github.com/user-attachments/assets/578c660a-ad3d-4477-af1d-17b60ca88fa1" />